pkg/agent/controller/agent.go

```diff
@@ -39,7 +39,7 @@ import (
 )
 
 const (
-	submarinerIpamGlobalIp = "submariner.io/globalIp"
+	submarinerIpamGlobalIP = "submariner.io/globalIp"
```

Comment:
```suggestion
	submarinerIPAMGlobalIP = "submariner.io/globalIp"
```

Reply:
I am changing only those that are flagged by the linter.

Reply:
I don't know why the linter didn't flag this - it should have. I think we should change it anyway.

Reply:
The linter has a list of acronyms, which doesn’t include “IPAM”. However while we’re changing names I think we should apply the rule consistently.

Reply:
oh I thought you were referring to _IP_ - that should be flagged and I see that @Jaanki did change that. We can change _Ipam_ -> _IPAM_  although in submariner we left it as _Ipam_.

Reply:
Yes. In submariner, we left it as it is suggesting that we add a custom linter to check for other such acronyms. Adding a custom linter requires some work which is not a priority IMO. So keeping it unchanged for the sake of consistency.